frontend/src/components/worksheets/WorksheetItemList.js

```diff
@@ -197,30 +196,25 @@ class WorksheetItemList extends React.Component {
                 },
             ];
         }
-        let focusedForButtonsItem;
+        let focusedItem;
         if (info && info.blocks.length > 0) {
             var worksheet_items = [];
             info.blocks.forEach(
                 function(item, index) {
                     const focused = index === this.props.focusIndex;
 
-                    // focusedForButtons determines whether clicking on Cell/Upload/Run will
+                    // focused determines whether clicking on Cell/Upload/Run will
                     // apply to this cell. If nothing is focused (focusIndex = -1),
-                    // append to the end by default.
-                    const focusedForButtons =
```

Comment:
This originally was used for appending to the end, since we are now prepending to the top of worksheet, no need for this anymore